scripts/asm-installer/install_asm

```diff
@@ -867,6 +877,15 @@ install_asm(){
   sleep 1
   info "...done!"
 
+  if [[ "$DISABLE_CANONICAL_SERVICE" != 1 ]]; then
```

Comment:
`if [[ "${DISABLE_CANONICAL_SERVICE}" -ne 1 ]]`

Very intuitively, != and = are string comparison operators in shell, you use -eq and -ne for numeric comparisons.

https://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-11.html

Reply:
Will change.